crane/config.go

```diff
@@ -219,6 +227,20 @@ func (c *config) initialize() {
 	}
 }
 
+// Override configuration
+func (c *config) overrideWithGlobalFlags() {
+	// At the moment, only `--tag` flag triggers override
+	if len(c.Tag()) == 0 {
+		return
+	}
+
+	for _, container := range c.containerMap {
+		container.OverrideImageTag(c.Tag())
+	}
+
+	os.Setenv("CRANE_TAG_FLAG", c.Tag())
```

Comment:
Thaｔ’s a good idea!  I totally agree with you. Maybe I can make an update within the next week.